pkg/stockings/iex.go

```diff
@@ -4,9 +4,10 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"github.com/shopspring/decimal"
 	"net/http"
 	"time"
+
+	"github.com/shopspring/decimal"
```

Comment:
is this go fmt'd as well?

Reply:
think so, not too sure whats wrong with my  go fmt